starter_code/express-spotify-app/app.js

```diff
@@ -0,0 +1,51 @@
+var SpotifyWebApi = require("spotify-web-api-node");
+// dvar spotifyAPI = new SpotifyWebApi();
+const express = require("express");
+const app = express();
+app.set("views", __dirname + "/views");
+app.set("view engine", "ejs");
+
+// Remember to paste here your credentials
+var clientId = "8b516eb922b640b286643f110c8053ba",
+  clientSecret = "8c96e275e34842219a928ef24d076a49";
+
+var spotifyApi = new SpotifyWebApi({
+  clientId: clientId,
+  clientSecret: clientSecret
+});
+
+app.get("/", (req, res, next) => {
+  res.render("index");
+});
+
+app.get("/artists", (req, res, next) => {
+  let artist = req.query.artist;
+
+  spotifyApi
+    .searchArtists(artist)
+    .then(artist => {
```

Comment:
I would name the callback argument something else that `artist` to prevent confusion with the `artist` variable you defined a few lines up